test/wilson/dom_test.cljs

```diff
@@ -114,12 +141,12 @@
                                    "j" {:class "warning"}
                                    "r" {:class "success"}
                                    {:class "info"}))})
-           [:table {:class "table table-hover"}
+           [:table {:class "table"}
```

Comment:
Why would we want to enforce a class in wilson? It is configurable isn't it? I thought the intent is to use `with-class` in cases like this. I'll update shrieker once this gets merged